### ISSUE TITLE
App Docker Tag 1.5.3 seems to be the worker?

### ISSUE THREAD

**Issue by frenos** [ciur/papermerge#241]
**Description**
Updating my Production System to 1.5.3 does not work. Changing the tag from `eugenci/papermerge:1.5.0` to `eugenci/papermerge:1.5.3` runs the worker, not the main app. Changing back to 1.5.0 makes it work as before, 1.5.2 seems to work too.

**Expected**
`eugenci/papermerge:1.5.3` should start the main web-app as before

**Actual**
see above

**Comment by hactar** [comment]
I also can't get 1.5.3 to start via docker - and as it has upgrade the db structure, I can't downgrade now either... :/ 

Log:

```
papermerge_app | [2020-12-19 22:12:43,669: INFO/Beat] Scheduler: Sending due task txt2db (papermerge.core.management.commands.worker.txt2db)
papermerge_app | [2020-12-19 22:12:48,009: CRITICAL/MainProcess] Unrecoverable error: ChannelError(None, "Cannot read file '/tmp/418640535_3c34bdbc-7532-4582-a694-a7e0f8f579ec.celery.msg' from queue.", None, '')
papermerge_app | Traceback (most recent call last):
papermerge_app |   File "/opt/app/.venv/lib/python3.8/site-packages/kombu/transport/filesystem.py", line 109, in _get
papermerge_app |     f = open(filename, 'rb')
papermerge_app | FileNotFoundError: [Errno 2] No such file or directory: '/tmp/418640535_3c34bdbc-7532-4582-a694-a7e0f8f579ec.celery.msg'
papermerge_app |
papermerge_app | During handling of the above exception, another exception occurred:
papermerge_app |
papermerge_app | Traceback (most recent call last):
papermerge_app |   File "/opt/app/.venv/lib/python3.8/site-packages/celery/worker/worker.py", line 205, in start
papermerge_app |     self.blueprint.start(self)
papermerge_app |   File "/opt/app/.venv/lib/python3.8/site-packages/celery/bootsteps.py", line 119, in start
papermerge_app |     step.start(parent)
papermerge_app |   File "/opt/app/.venv/lib/python3.8/site-packages/celery/bootsteps.py", line 369, in start
papermerge_app |     return self.obj.start()
papermerge_app |   File "/opt/app/.venv/lib/python3.8/site-packages/celery/worker/consumer/consumer.py", line 318, in start
papermerge_app |     blueprint.start(self)
papermerge_app |   File "/opt/app/.venv/lib/python3.8/site-packages/celery/bootsteps.py", line 119, in start
papermerge_app |     step.start(parent)
papermerge_app |   File "/opt/app/.venv/lib/python3.8/site-packages/celery/worker/consumer/consumer.py", line 596, in start
papermerge_app |     c.loop(*c.loop_args())
papermerge_app |   File "/opt/app/.venv/lib/python3.8/site-packages/celery/worker/loops.py", line 113, in synloop
papermerge_app |     connection.drain_events(timeout=2.0)
papermerge_app |   File "/opt/app/.venv/lib/python3.8/site-packages/kombu/connection.py", line 324, in drain_events
papermerge_app |     return self.transport.drain_events(self.connection, **kwargs)
papermerge_app |   File "/opt/app/.venv/lib/python3.8/site-packages/kombu/transport/virtual/base.py", line 963, in drain_events
papermerge_app |     get(self._deliver, timeout=timeout)
papermerge_app |   File "/opt/app/.venv/lib/python3.8/site-packages/kombu/utils/scheduling.py", line 56, in get
papermerge_app |     return self.fun(resource, callback, **kwargs)
papermerge_app |   File "/opt/app/.venv/lib/python3.8/site-packages/kombu/transport/virtual/base.py", line 1001, in _drain_channel
papermerge_app |     return channel.drain_events(callback=callback, timeout=timeout)
papermerge_app |   File "/opt/app/.venv/lib/python3.8/site-packages/kombu/transport/virtual/base.py", line 745, in drain_events
papermerge_app |     return self._poll(self.cycle, callback, timeout=timeout)
papermerge_app |   File "/opt/app/.venv/lib/python3.8/site-packages/kombu/transport/virtual/base.py", line 402, in _poll
papermerge_app |     return cycle.get(callback)
papermerge_app |   File "/opt/app/.venv/lib/python3.8/site-packages/kombu/utils/scheduling.py", line 56, in get
papermerge_app |     return self.fun(resource, callback, **kwargs)
papermerge_app |   File "/opt/app/.venv/lib/python3.8/site-packages/kombu/transport/virtual/base.py", line 405, in _get_and_deliver
papermerge_app |     message = self._get(queue)
papermerge_app |   File "/opt/app/.venv/lib/python3.8/site-packages/kombu/transport/filesystem.py", line 115, in _get
papermerge_app |     raise ChannelError(
papermerge_app | amqp.exceptions.ChannelError: Cannot read file '/tmp/418640535_3c34bdbc-7532-4582-a694-a7e0f8f579ec.celery.msg' from queue.
papermerge_app | [2020-12-19 22:12:48,555: INFO/MainProcess] beat: Shutting down...
```

**Comment by hactar** [comment]
PS: if anyone is stuck, you can run docker-compose build app first to build the app image yourself, and then relaunch papermerge via docker-compose, this will get you pack into a running state.

**Comment by ciur** [comment]
> Updating my Production System to 1.5.3 does not work. Changing the tag from eugenci/papermerge:1.5.0 to eugenci/papermerge:1.5.3 runs the worker, not the main app. Changing back to 1.5.0 makes it work as before, 1.5.2 seems to work too.

Yes, that's correct. By mistake I pushed app image onto the worker. I will fix it this week as I will publish 1.5.4 release.

**Comment by ciur** [comment]
release 1.5.4 is out. Please use it, instead of 1.5.3 which has above mentioned issues.
